pkg/urbit/vere/pier.c

```diff
@@ -1066,14 +1088,31 @@ _pier_work_poke(void*   vod_p,
         u3_writ* wit_u = _pier_writ_find(pir_u, evt_d);
 
         if ( !wit_u ) {
-          fprintf(stderr, "poke: no writ: %" PRIu64 "\r\n", evt_d);
+          u3l_log("poke: no writ: %" PRIu64 "\r\n", evt_d);
           goto error;
         }
         _pier_work_complete(wit_u, mug_l, u3k(r_jar));
       }
       break;
     }
 
+    case c3__stdr: {
+      if ( (c3n == u3r_trel(jar, 0, &p_jar, &q_jar)) ||
+           (c3n == u3ud(p_jar)) ||
+           (u3r_met(6, p_jar) != 1) ||
```

Comment:
This condition fails on the `worker: play $N` printf -- as it will for any others that occurs before receiving the first event. `u3r_met(6, p_jar) > 1` will allow for such prints (at "event 0").